bin/upload-config-to-heroku.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/* Take a taskcluster.base.config config file and generate a command that will
+   upload all values to heroku */
```

Comment:
It's helpful for me as I do development locally.  I think it's a generally useful script because it lets you mirror your local environment on a heroku instance easily.  If you're against it, I can remove it from here.